package.json

```diff
@@ -86,7 +87,7 @@
     "basic-auth": "^2.0.1",
     "cookie": "^0.4.0",
     "core-util-is": "^1.0.2",
-    "elastic-apm-http-client": "^9.9.0",
+    "elastic-apm-http-client": "elastic/apm-nodejs-http-client#trentm/truncate-in-chars-not-bytes",
```

Comment:
I presume the intention is to land this PR after the client changes land and that the github repo and `XXX` will be swapped out.

Reply:
Yup, done.